src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { TextField } from '@mui/material';
+
+function CustomTextField(props) {
+
+//    const handleInputsChanged = (e) => {
+//         if(e.id) {
+
+//         }
+//    }
+ 
+  return (
+    <div>
+        <TextField 
+            id={props.id}
+            name={props.name}
+            type={props.type}
+            label={props.label}
+            // placeholder="example@example.com"
+            onChange={handleInputsChanged}
+            // helperText="Please enter a valid e-mail address"
+            // required
+            // error={hasEmailError}
+            // fullWidth    
+        />
+    </div>
+  )
+}
+
+export default TextField
```

Comment:
We should decide what we want to do with this code. The idea here was to create a custom-styled TextField and re-use it in places where there is a text field.